### ISSUE TITLE
Minor: Found site where the TransOver does not work

### ISSUE THREAD

**Issue by mrkeuz** [artemave/translate_onhover#96]
Hi thanks again for awesome extension, it fast, light and very usable! Just awesome! 🚀

Found a site, where the TransOver does not work:
https://blogs.oracle.com/javamagazine/post/java-inheritance-composition 

After several refreshes it finally starts works (only once), but I understand why and what difference versus previous page refreshes. Further I could not reproduce it (I mean NORMAL work of translation) and the translation still didn't work.

Think it is because messy html/js code behind. Maybe you might be interested to investigate this issue. By the way, this is the first and so far the only site in a long time, where TransOver does not work. So issue very minor.

Thanks again! 🚀🚀🚀

### Env
TransOver 1.63

Tested in (also in private mode):
- Brave1.41.100 Chromium: 103.0.5060.134
- Firefox 102.0 (64-bit)

**Comment by artemave** [comment]
Indeed. No errors in dev console - just nothing happens. Intriguing. Thanks for reporting!
